x-pack/plugins/observability_solution/observability/public/components/alerts_table/common/render_cell_value.tsx

```diff
@@ -71,63 +69,66 @@ const getRenderValue = (mappedNonEcsValue: any) => {
-      case ALERT_SEVERITY:
-        return <AlertSeverityBadge severityLevel={value ?? undefined} />;
-      case ALERT_EVALUATION_VALUE:
-        const valuesField = getMappedNonEcsValue({
-          data,
-          fieldName: ALERT_EVALUATION_VALUES,
-        });
-        const values = getRenderValue(valuesField);
-        return valuesField ? values : value;
-      case ALERT_REASON:
-        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
-        const alert = parseAlert(observabilityRuleTypeRegistry)(dataFieldEs);
+  switch (columnId) {
+    case ALERT_STATUS:
+      if (value !== ALERT_STATUS_ACTIVE && value !== ALERT_STATUS_RECOVERED) {
+        // NOTE: This should only be needed to narrow down the type.
+        // Status should be either "active" or "recovered".
+        return null;
+      }
+      return <AlertStatusIndicator alertStatus={value} />;
+    case TIMESTAMP:
+      return <TimestampTooltip time={new Date(value ?? '').getTime()} timeUnit="milliseconds" />;
+    case ALERT_DURATION:
+      return asDuration(Number(value));
+    case ALERT_SEVERITY:
+      return <AlertSeverityBadge severityLevel={value ?? undefined} />;
+    case ALERT_EVALUATION_VALUE:
+      const valuesField = getMappedNonEcsValue({
+        data,
+        fieldName: ALERT_EVALUATION_VALUES,
+      });
+      const values = getRenderValue(valuesField);
+      return valuesField ? values : value;
+    case ALERT_REASON:
+      const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+      if (!observabilityRuleTypeRegistry) return <>{value}</>;
+      const alert = parseAlert(observabilityRuleTypeRegistry)(dataFieldEs);
 
-        return (
-          <EuiLink
-            data-test-subj="o11yGetRenderCellValueLink"
-            css={{ display: 'contents' }}
-            onClick={() => setFlyoutAlert && setFlyoutAlert(alert.fields[ALERT_UUID])}
-          >
-            {alert.reason}
-          </EuiLink>
-        );
-      case ALERT_RULE_NAME:
-        const ruleCategory = getMappedNonEcsValue({
-          data,
-          fieldName: ALERT_RULE_CATEGORY,
-        });
-        const tooltipContent = getRenderValue(ruleCategory);
-        return <CellTooltip value={value} tooltipContent={tooltipContent} />;
-      default:
-        return <>{value}</>;
-    }
-  };
+      return (
+        <EuiLink
+          data-test-subj="o11yGetRenderCellValueLink"
+          css={{ display: 'contents' }}
+          onClick={() => setFlyoutAlert && setFlyoutAlert(alert.fields[ALERT_UUID])}
+        >
+          {alert.reason}
+        </EuiLink>
+      );
+    case ALERT_RULE_NAME:
+      const ruleCategory = getMappedNonEcsValue({
+        data,
+        fieldName: ALERT_RULE_CATEGORY,
+      });
+      const tooltipContent = getRenderValue(ruleCategory);
+      return <CellTooltip value={value} tooltipContent={tooltipContent} />;
+    default:
+      return <>{value}</>;
+  }
```

Comment:
Just for reference, seems to be no substantive differences between these blocks, just indentation change.

Reply:
the `react.memo` is doing that I think

Reply:
Yeah something above changed the indentation, it happens. I just wanted to flag it for whoever on my team takes a look at this tomorrow. 👍